concent_api/core/transfer_operations.py

```diff
@@ -191,12 +193,12 @@ def send_request_to_cluster_storage(headers, request_http_address):
     )
 
 
-def sign_message(golem_message, priv_key, pub_key):
+def sign_message(golem_message, priv_key):
```

Comment:
this function deosn't belong here; please move it to `utils/helpers.py`

Reply:
Done